<html lang='en'><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Rapid adaptation of the foodborne pathogen Campylobacter jejuni</title></head><body><h1>Rapid adaptation of the foodborne pathogen Campylobacter jejuni</h1><p>Rapid adaptation of the foodborne pathogen Campylobacter jejuni</p><p>&nbsp;&nbsp;&nbsp;&nbsp;By Azam Ali Sher | Michigan State University</p><p><em>&nbsp;&nbsp;&nbsp;Campylobacter</em></p><p><em>&nbsp;&nbsp;&nbsp;&nbsp;jejuni</em>&nbsp;&nbsp;&nbsp;is a leading cause of foodborne diarrheal infections in the USA (Scallan et al., 2011).The Centers for Disease Control and Prevention (CDC) estimates there are around 1.5 million human cases of campylobacteriosis reported annually in the USA and $270 million in direct medical costs (CDC., 2019).A recent report on drug-resistant pathogens from the CDC revealed that 28% of all</p><p><em>&nbsp;&nbsp;&nbsp;Campylobacter</em>&nbsp;&nbsp;&nbsp;isolates in the USA were drug-resistant (CDC., 2019).Thus, Campylobacter has become a significant threat to public health, which drives the need to examine its survival and evolutionary mechanisms in all sources of this pathogen.</p><p><em>&nbsp;&nbsp;&nbsp;Campylobacter</em>&nbsp;&nbsp;&nbsp;can survive in many different places including the gastrointestinal tracts of humans and many animals as well as in environmental sites.Evolution of pathogenic bacteria permits them to survive in novel environments.In fact, without insight into pathogen evolution mechanisms, it is difficult to understand disease transmission, pathogenesis, and the emergence of antibiotic resistance.However, many previous evolution studies were performed using laboratory-adapted bacterial strains rather than clinically relevant pathogens such as</p><p><em>&nbsp;&nbsp;&nbsp;C. jejuni</em>&nbsp;&nbsp;&nbsp;.</p><p>&nbsp;&nbsp;&nbsp;Our research team, led by Dr Linda Mansfield at Michigan State University, conducted an evolution study to understand the adaptation of host-adapted</p><p><em>&nbsp;&nbsp;&nbsp;C. jejuni</em>&nbsp;&nbsp;&nbsp;to growth in the novel environment of nutrient-enriched laboratory growth medium (Sher et al., 2020).We used a serial transfer study design where five independent highly motile mouse gut adapted populations of</p><p><em>&nbsp;&nbsp;&nbsp;C. jejuni</em>&nbsp;&nbsp;&nbsp;were passaged individually in the lab for 35 days.Every 24 hours, 100uL culture of each bacterial population was transferred to a fresh growth medium.</p><p>&nbsp;&nbsp;&nbsp;We were interested in learning how a motile pathogenic bacterium such as</p><p><em>&nbsp;&nbsp;&nbsp;C. jejuni</em>&nbsp;&nbsp;&nbsp;evolves in nutrient-rich media compared to the complex environment of the mammalian gut, where motility is required to colonize and survive.A semi-solid agar motility assay was used to observe changes in the motility of the bacterium.Three motility phenotypes of</p><p><em>&nbsp;&nbsp;&nbsp;C. jejuni</em>&nbsp;&nbsp;&nbsp;were observed in each of the five populations we studied: reversibly non-motile, irreversibly non-motile, and motile.However, the irreversibly non-motile phenotype dominated after day 20 in these five independent evolved populations.By day 5, a significant proportion of colonies in all five populations were found to be non-motile.This loss of motility increased over time, and about 80 percent of all five independent populations were non-motile by day 35.Electron microscopy revealed a total loss of flagella in one evolved population compared to the initial inoculum.</p><p>&nbsp;&nbsp;&nbsp;To understand the genomic changes behind this motility loss, the authors performed whole-genome re-sequencing on all five evolved populations.Many disruptive mutations in genes essential for the flagellar biosynthesis cascade were detected, particularly in</p><p><em>&nbsp;&nbsp;&nbsp;rpo</em>&nbsp;&nbsp;&nbsp;N,</p><p><em>&nbsp;&nbsp;&nbsp;Cj</em>&nbsp;&nbsp;&nbsp;0390,</p><p><em>&nbsp;&nbsp;&nbsp;mot</em>&nbsp;&nbsp;&nbsp;A,</p><p><em>&nbsp;&nbsp;&nbsp;flg</em>&nbsp;&nbsp;&nbsp;R,</p><p><em>&nbsp;&nbsp;&nbsp;flg</em>&nbsp;&nbsp;&nbsp;S, and</p><p><em>&nbsp;&nbsp;&nbsp;flh</em>&nbsp;&nbsp;&nbsp;A (see Burnham et al., 2020).Especially notable were large deletions in the gene</p><p><em>&nbsp;&nbsp;&nbsp;rpo</em>&nbsp;&nbsp;&nbsp;N (sigma 54) and the downstream region, which in</p><p><em>&nbsp;&nbsp;&nbsp;C. jejuni</em>&nbsp;&nbsp;&nbsp;strain 11168 contains a number of pseudogenes.PCR assays showed that</p><p><em>&nbsp;&nbsp;&nbsp;rpo</em>&nbsp;&nbsp;&nbsp;N and</p><p><em>&nbsp;&nbsp;&nbsp;Cj</em>&nbsp;&nbsp;&nbsp;0390 mutants accumulated over time.</p><p><em>&nbsp;&nbsp;&nbsp;rpo</em>&nbsp;&nbsp;&nbsp;N mutants grew markedly faster than the initial motile strain in the novel environment.However, to confirm which mutation triggered the initial loss of motility required another comprehensive evolution study, including genomic and transcriptomic analyses.After observing phenotypic motility loss and then genomic disruption of flagellar biosynthesis cascade, the authors also performed scanning transmission electron microscopy that revealed a total loss of flagella in evolved populations compared to the initial inoculum.</p><p>&nbsp;&nbsp;&nbsp;While</p><p><em>&nbsp;&nbsp;&nbsp;in vitro</em>&nbsp;&nbsp;&nbsp;based studies should be interpreted cautiously, Dr Sher, the first author of the study, commented that 'Experimental evolution studies could be an effective tool to discover novel drug targets and vaccine candidates against pathogenic bacteria like</p><p><em>&nbsp;&nbsp;&nbsp;C. jejuni.</em>&nbsp;&nbsp;&nbsp;'</p><p>&nbsp;&nbsp;&nbsp;Dr Bell, a leading author of the study, mentioned, 'We did not expect the large chromosomal deletions in the</p><p><em>&nbsp;&nbsp;&nbsp;rpo</em>&nbsp;&nbsp;&nbsp;N region, though those deletions are consistent with the removal of selection pressure to maintain motility in the host, the expendability of downstream pseudogenes, and the likely history of genome size reduction in this pathogen.'</p><p>&nbsp;&nbsp;&nbsp;Dr Mansfield, the corresponding author, stated, 'This work indicates that</p><p><em>&nbsp;&nbsp;&nbsp;C. jejuni</em>&nbsp;&nbsp;&nbsp;undergoes genome reduction when grown in rich media and suggests that its evolutionary mechanisms in different hosts and environmental niches should be explored.'</p><p>&nbsp;&nbsp;&nbsp;program mediated by diverse two-component signal transduction systems and basal flagellar proteins is broadly conserved in polar flagellates.</p><p><em>&nbsp;&nbsp;&nbsp;MBio</em>&nbsp;&nbsp;&nbsp;11, 1–25.doi:10.1128/mBio.03107-19.</p><p>&nbsp;&nbsp;&nbsp;CDC.(2019).Antibiotic resistance threats in the United States, 2019.</p><p><em>&nbsp;&nbsp;&nbsp;U.S. Dep.Heal.Hum.Serv.CDC; 2019.</em>&nbsp;&nbsp;&nbsp;, 114.doi:CS239559-B.</p><p>&nbsp;&nbsp;&nbsp;Scallan, E., Hoekstra, R. M., Angulo, F.J., Tauxe, R. V, Widdowson, M.A., Roy, S.L., et al.(2011).Foodborne illness acquired in the United States-Major pathogens.</p><p><em>&nbsp;&nbsp;&nbsp;Emerg.Infect.Dis.</em>&nbsp;&nbsp;&nbsp;17, 7–15.doi:10.3201/eid1701.P11101.</p><p>&nbsp;&nbsp;&nbsp;Original article: Experimental Evolution of Campylobacter jejuni Leads to Loss of Motility, rpoN (σ54) Deletion and Genome Reduction</p><p>&nbsp;&nbsp;&nbsp;REPUBLISHING GUIDELINES: Open access and sharing research is part of Frontiers’ mission.Unless otherwise noted, you can republish articles posted in the Frontiers news blog — as long as you include a link back to the original research.Selling the articles is not allowed.</p><p><p><strong><i>Posted On: </i></strong>2021-01-04T00:00:00<br></p></p><p><p><img src='http://localhost:8083/blackboxP3/jpg/L0C04F2SS.jpg' style='max-width: 480px;'/></p><p><strong>Content Date:</strong>&nbsp;2021-01-04</p><p><strong>Download Date:</strong>&nbsp;2021-07-10</p><p><strong>Document ID:</strong>&nbsp;L0C04F2SO</p></body></html>